book/src/user/mining-docker.md

```diff
@@ -0,0 +1,33 @@
+# Mining with Zebra in Docker
+
+All [published images](https://hub.docker.com/r/zfnd/zebra/tags) containing the
+`.experimental` suffix are compiled with the `getblocktemplate-rpcs` feature, so
+you can use them for your mining operations. The images are pre-configured for
+Testnet, and you must pass your rewards address as an environment variable to
+the container when starting it. For example, executing
+
+```bash
+docker run -e MINER_ADDRESS="t27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v" -p 18232:18232 zfnd/zebra:v1.0.1.experimental
```

Comment:
```suggestion
docker run -e MINER_ADDRESS="t27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v" -p 18232:18232 zfnd/zebra:v1.1.0.experimental
```

Reply:
Addressed in [f4ec740](https://github.com/ZcashFoundation/zebra/pull/7179/commits/f4ec740f302ba0a72752c64a0939538d9b533bf9).